chromadb/api/models/AsyncCollection.py

```diff
@@ -91,7 +100,7 @@ async def get(
         limit: Optional[int] = None,
         offset: Optional[int] = None,
         where_document: Optional[WhereDocument] = None,
-        include: Include = ["metadatas", "documents"],
+        include: Include = ["metadatas", "documents"],  # type: ignore[list-item]
```

Comment:
I am a bit confused - I don't see this type error'ing on my machine?

Reply:
Likely a mypy version discrepancy